servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/NoopAsyncContextProvider.java

```diff
@@ -252,7 +252,7 @@ public ContextMap copy() {
 
         @Override
         public int hashCode() {
-            return super.hashCode();
+            return System.identityHashCode(this);
```

Comment:
Not a big thing, but I wonder why calling `identityHashCode` is better than `hashCode` on the parent in those cases?

Reply:
It was complaining with `UselessOverridingMethod` warning when it used `super.hashCode()` but if I remove it, it was complaining that this class overrides `equals` but does not override `hashCode`. So, I had to use `System.identityHashCode(this)` to keep the override but fool the checker.

Reply:
Ah I see - still weird but ok heh.